### PR TITLE
Db filling

### PR DESCRIPTION
**Pull request by Andrzej97**

### CODE REVIEW

File: src/common_utils/database_preparing/set_init_configure.py [hunk 1]
```diff
@@ -28,23 +16,31 @@ def initialize_language_utils_database(db):
...
         db.add_many_new_docs_to_collection('polish_stop_words', list)
 
 
 def from_txt_file_to_list(path):
-    file = open(path, "r")
+    file = open(path, mode='r', encoding='utf-8')
     lines = list(map(lambda x: x.rstrip(), list(file.readlines())))
     return lines
 
+def create_responses_collection(db):
```
Comment (v-tec2706) [inline src/common_utils/database_preparing/set_init_configure.py]: can be removed even now, the creation of all collections is handled on my branch.

File: src/common_utils/language_utils/sentence_filter_utils.py [hunk 2]
```diff
@@ -94,4 +106,89 @@ def extract_lemmas_and_filter_stopwords(self, sentence):
...
-        return list(filter(lambda x: x is not None, lemmas))
\ No newline at end of file
+        return lemmas
+
+    def extract_complex_lemmas_and_filter_stopwords(self, phrase):
+        analysis = self.utils.morfeusz.analyse(phrase)
+        tags = set([])
+        old_word_index = 0
+        single_tag = set([])
+        #print('ANALYSIS RESULT:\n', analysis)
```
Comment (v-tec2706) [inline src/common_utils/language_utils/sentence_filter_utils.py]: we don't need them on master xD

File: src/common_utils/language_utils/sentence_filter_utils.py [hunk 3]
```diff
@@ -94,4 +106,89 @@ def extract_lemmas_and_filter_stopwords(self, sentence):
...
+    def generate_filtered_words_lemmas_combinations(self, complex_lemmas_list):
+        comb_idx = 0
+        combinations_dict = {}
+
+        def create_lemmas_combinations(complex_lemmas_list, start_idx):
+            nonlocal comb_idx
+            if len(complex_lemmas_list) == 0:  # end of recursion
+                combinations_dict[comb_idx] = []
+                comb_idx = comb_idx + 1
+                #print("Comb_idx = ", comb_idx, ", dict = ", combinations_dict)
```
Comment (v-tec2706) [inline src/common_utils/language_utils/sentence_filter_utils.py]: these ones as well :P

File: src/common_utils/language_utils/sentence_filter_utils.py [hunk 4]
```diff
@@ -94,4 +106,89 @@ def extract_lemmas_and_filter_stopwords(self, sentence):
...
+        if len(single_tag) > 0:
+            single_tags_list = list(single_tag)
+            single_tags_list.sort()
+            tags.add(list_to_str_with_colons(single_tags_list))
+        return tags
+
+    def is_stopword(self, word):
+        return word.lower() in self.stop_words
+
+    def split_to_norm_and_complex_lemmas(self, lemmas_list):
```
Comment (rpych) [inline src/common_utils/language_utils/sentence_filter_utils.py]: this is from my branch with algorithm improvement, this is probably not used with Your changes, may be removed

File: src/common_utils/language_utils/sentence_filter_utils.py [hunk 5]
```diff
@@ -94,4 +106,89 @@ def extract_lemmas_and_filter_stopwords(self, sentence):
...
+            splitted_lemmas = lemma.split(':')
+            if len(splitted_lemmas) > 1 and splitted_lemmas[1] not in ['s', 's1', 's2', 's3', 'v1', 'v2', 'v3']:
+                complex_lemmas.append(lemma)
+            elif len(splitted_lemmas) > 1:
+                normal_lemmas.append(splitted_lemmas[0])
+            else:
+                normal_lemmas.append(lemma)
+        return normal_lemmas, complex_lemmas
+
+    def generate_filtered_words_lemmas_combinations(self, complex_lemmas_list):
```
Comment (rpych) [inline src/common_utils/language_utils/sentence_filter_utils.py]: this is from my branch with algorithm improvement, this is probably not used with Your changes, may be removed

File: src/common_utils/language_utils/sentence_filter_utils.py [hunk 6]
```diff
@@ -94,4 +106,89 @@ def extract_lemmas_and_filter_stopwords(self, sentence):
...
+                create_lemmas_combinations(lemma_param, comb_idx)
+
+                for k in range(start_idx, comb_idx):
+                    combinations_dict[k].append(possibility)
+            #print("DICT:", combinations_dict)
+
+        create_lemmas_combinations(complex_lemmas_list, 0)
+        return combinations_dict
+
+# input = "wydziały-i-podstawowe-jednostki-organizacyjne"
```
Comment (rpych) [inline src/common_utils/language_utils/sentence_filter_utils.py]: nice to be removed

File: src/common_utils/language_utils/sentence_filter_utils.py [hunk 7]
```diff
@@ -94,4 +106,89 @@ def extract_lemmas_and_filter_stopwords(self, sentence):
...
+# # print('input: ' + input)
+# sentence_filtered = SentenceFilter().filter_sentence(input, ['noun'])
+# print('output: ')
+# for sentence in sentence_filtered:
+#     print("    " + sentence)
+# #
+# #
+# # print(SentenceFilter().extract_lemma('wydziały'))
+# print(SentenceFilter().extract_lemmas_and_filter_stopwords(input))
```
Comment (rpych) [inline src/common_utils/language_utils/sentence_filter_utils.py]: nice to be removed

**Pull request merged**